Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM alpine:3.16.2
 COPY ./build/linux/jx-ui /usr/bin/jx-ui
 COPY ./web/build web/build
+COPY ./web/.env-prod web/.env
```

Comment:
Doesn't sveltekit automatically set this during build?

Reply:
I don't know

Reply:
can u try building the image without that line and see if it's set?

Reply:
heh. There is no .env file anywhere in the image. Looks like it must be copied in, but the path I tried above, is not correct.